### DISCUSSION TITLE
v0.1.3

### DISCUSSION THREAD

**Discussion by DominusKelvin** [sailscastshq/boring-stack#83] [Announcements]
### inertia-sails@v0.1.8
* [feat] implement handling validation errors and flash messages by @DominusKelvin in https://github.com/sailscastshq/boring-stack/pull/80
### mellow-vue@0.1.3
* Handle validation errors on login and signup pages

**Full Changelog**: https://github.com/sailscastshq/boring-stack/compare/v0.1.2...v0.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/sailscastshq/boring-stack/releases/tag/v0.1.3'>v0.1.3</a>.</em>

**Comment by DominusKelvin** [comment]
The docs now have [Validation](https://docs.sailscasts.com/boring-stack/validation) and [Flash messages](https://docs.sailscasts.com/boring-stack/flash-messages) pages to explain how to use them in The Boring Stack. Also `mellow-vue` has been updated for login and signup pages to handle errors.

**Comment by lennyAiko** [comment]
I love the updates you've done. What were the bugs you noticed with flash messages?

I will have to pull mellow-vue and go through the new work you've added.
